azure/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM centos:latest
+RUN yum -y install WALinuxAgent && yum -y clean all
+COPY exports /
```

Comment:
The current syntax is copying a file called exports into the container's root, which you don't have included. I think for the time being what you want is `COPY service.template /exports/`